imports/client/components/NotificationCenter.tsx

```diff
@@ -220,6 +242,7 @@ const GuessMessage = React.memo(({
   return (
     <Toast onClose={dismissGuess}>
       <Toast.Header>
+        <StyledGuessNotificationHeader>
         <StyledGuessHeader>
```

Comment:
The linter insists you indent the children of `StyledGuessNotificationHeader` here.